.github/workflows/connect-dev-release.yml

```diff
@@ -0,0 +1,57 @@
+name: "[Build/Test] connect-explorer"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  pull_request:
+    paths:
+      - "packages/connect*/**"
+
+env:
+  DEV_SERVER_URL: "https://dev.suite.sldev.cz"
+
+jobs:
+  build-connect-explorer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
let's update it to `aws-actions/configure-aws-credentials@v4`

Reply:
Done